### ISSUE TITLE
Re-creating Track.AdditionalFields causes tag duplication

### ISSUE THREAD

**Issue by GOATS2K** [Zeugma440/atldotnet#260]
## The problem

Tags written with `Track.AdditionalFields` get duplicated upon re-saving with changes.

## Environment

* ATL version (or git revision) that exhibits the issue: 5.21.0
* Last ATL version that did not exhibit the issue (if applicable): none afaik
* OS/version used to run ATL: Windows 10 22H2 19045

## Code To Reproduce Issue [ Good To Have ]

Consider this scenario, a track initially gets tagged like this:

```csharp
track.AdditionalFields = new Dictionary<string, string>()
        {
            { "CATALOGNUMBER",  "CAT1" }
        };
track.Save();
```

Then, the user updates the catalog number:
```csharp
track.AdditionalFields = new Dictionary<string, string>()
        {
            { "CATALOGNUMBER",  "CAT2" }
        };
track.Save();
```

Observe that the `CATALOGNUMBER` field has been duplicated in the file (flac), and contains both `CAT1` and `CAT2`.

**Comment by GOATS2K** [comment]
Aha, I just realized that Track already has `ISRC` and `CatalogNumber` tags - using those instead bypasses the duplication issue. I still think this'd be good to fix though. :)

**Comment by Zeugma440** [comment]
Fix is available on today's v5.22.

Please close the issue if it works on your side~
